textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
I want to hear the opinions of @JavierJia and team 1 ( @sandeepreddy602 ) on this change.